dashboard/app/models/external.rb

```diff
@@ -23,6 +23,12 @@
 #
 
 class External < DSLDefined
+
+  # Check if the level has a hand-written submit button. Once all submit buttons are removed from markdown, this can go away.
+  def has_submit_button?
+    properties['markdown'].try(:include?, 'next-stage') && properties['markdown'].try(:include?, 'submitButton')
```

Comment:
That's a good point.  The markup we're looking for typically looks like this:
```html
<a class="btn btn-large btn-primary next-stage submitButton">Continue</a>
```
I could just search for `submitButton` but it seemed safer to check for the presence of both.  I could check for the full string (assuming it's always the same) at the risk of missing buttons with re-ordered classes.

Either way this method is temporary enough I think we can fix any false positives or negatives as they come up.